test/test_tsubaiso_sdk.rb

```diff
@@ -572,6 +581,13 @@ def test_list_reimbursements
     @api.destroy_reimbursement(reimbursement_b_id) if reimbursement_b_id
   end
 
+  def test_list_reimbursement_reason_masters
+    reimbursement_reason_masters_list = @api.list_reimbursement_reason_masters
+    assert_equal 200, reimbursement_reason_masters_list[:status].to_i, reimbursement_reason_masters_list.inspect
+    assert reimbursement_reason_masters_list[:json]
+    assert !reimbursement_reason_masters_list[:json].any?{ |x| x[:ccode] != 3}
```

Comment:
It seems  this test is different with another tests, but I couln't found another ways, since we didn't have create API action for now.